tests/deadlock1.sh

```diff
@@ -0,0 +1,9 @@
+set -e
+for i in {1..100} ; do
+    timeout 1s ./mimpirun 2 examples_build/deadlock1
+    timeout 1s ./mimpirun 2 examples_build/deadlock2
+    timeout 1s ./mimpirun 2 examples_build/deadlock3
+    timeout 1s ./mimpirun 3 examples_build/deadlock4
+    timeout 1s ./mimpirun 2 examples_build/deadlock5
+    timeout 1s ./mimpirun 2 examples_build/deadlock6
```

Comment:
Deadlock 6 wysyła dość dużo danych. Nie wiem czy jest po co robić komuś false positive tylko ze względu na to, że nie wyrobił się w czasie. Sugeruję większy timeout np. 5s.